src/components/Header.tsx

```diff
@@ -33,11 +33,14 @@ const Header: React.FC<HeaderProp> = ({ pathname }) => {
 	const [light, setLight] = useState(false);
 
 	useEffect(() => {
+<<<<<<< HEAD
```

Comment:
please look into this theonly change should be the addition to calculator route

Reply:
Yeah, i fixed it in the file.